contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -33,6 +33,7 @@
 import numbers
 import copy
 from collections import Iterable
+from fractions import Fraction
```

Comment:
I'm guessing the style in this file is to do the "from" style imports but I think I'd prefer it if moving forward we could leave the namespaces in... I still think thats more readable, especially for stuff like this.

Reply:
updated code to conform to style guidelines.